pkg/server/proto/server.proto

```diff
@@ -5094,6 +5122,89 @@ message ConfigSyncPipelineResponse {
   map<string, Ref.Pipeline> synced_pipelines = 1;
 }
 
+/********************************************************************
+* ProjectTemplate
+********************************************************************/
+
+message GetProjectTemplateRequest {
+  Ref.ProjectTemplate project_template = 1;
+}
+
+message GetProjectTemplateResponse {
+  // A single projecttemplate
+  ProjectTemplate project_template = 1;
+}
+
+message ListProjectTemplatesRequest {
+  PaginationRequest pagination = 1;
+}
+
+message ListProjectTemplatesResponse {
+  // The full list of projecttemplates
+  repeated ProjectTemplate project_templates = 1;
+  PaginationResponse pagination = 2;
+  uint64 total_count = 3;
+}
+
+message UpsertProjectTemplateRequest {
+  ProjectTemplate project_template = 1;
+}
+
+message UpsertProjectTemplateResponse {
+  ProjectTemplate project_template = 1;
+}
+
+message DeleteProjectTemplateRequest {
+  Ref.ProjectTemplate project_template = 1;
+}
+
+message DeleteProjectTemplateResponse {
+}
+
+message ProjectTemplate {
+  // Unique ID of the ProjectTemplate
+  string id = 1;
+
+  // Unique name of the ProjectTemplate
+  string name = 2;
+
+  // A brief summary for use by platform engineers to distinguish between
+  // multiple templates
+  string summary = 3;
+
+  // A long summary of what the ProjectTemplate is to be used for. This summary
+  // is shared between platform engineers and application developers.
+  string expanded_summary = 4;
+
+  // The template which is rendered when a Project is created from a
+  // ProjectTemplate to be shown to application developers.
+  string readme_template = 5;
+
+  // Settings for the Waypoint project that should be set when a project is
+  // created from a ProjectTemplate.
+  WaypointProject waypoint_project = 6;
```

Comment:
What kind of settings are we expecting here? Forgive me I don't recall them from the RFC